sylt-parser/src/statement.rs

```diff
@@ -302,6 +311,46 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
             (ctx, Use { path: path_ident, name: alias, file })
         }
 
+        // `from path/to/file use var`
+        [T::From, ..] => {
+            let ctx = ctx.skip(1);
+            let (ctx, path_ident) = path(ctx)?;
+            let path = &path_ident.name;
+            let name = path
+                .trim_start_matches("/")
+                .trim_end_matches("/")
+                .to_string();
+            let file = {
+                let parent = if path.starts_with("/") {
+                    ctx.root
+                } else {
+                    ctx.file.parent().unwrap()
+                };
+                // Importing a folder is the same as importing exports.sy
+                // in the folder.
+                parent.join(if path == "/" {
+                    format!("exports.sy")
+                } else if path_ident.name.ends_with("/") {
+                    format!("{}/exports.sy", name)
+                } else {
+                    format!("{}.sy", name)
+                })
+            };
+            let ctx = expect!(ctx, T::Use, "Expected 'use' after path");
```

Comment:
I like to mention what we expected to parse - so a clearer syntax error might be:
```suggestion
            let ctx = expect!(ctx, T::Use, "Expected 'use' after path when parsing from-import");
```